<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>welcome</class>
 <widget class="QWidget" name="welcome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">#frame_3{border-image: url(:/new/prefix1/snow2.png) 0 0 0 0 stretch stretch;}
#welcome{background-color: black}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QFrame" name="frame_3">
     <property name="styleSheet">
      <string notr="true">#pushButton_command{border-image: url(:/new/prefix1/pngwing.com.png) 0 0 0 0 stretch stretch; color : white;padding-bottom: 10px; height: 50}
#pushButton_command:pressed{color: black}
#pushButton_command:hover{border-style: outset; border-width: 2px; border-radius: 10px; border-color: beige;}
#pushButton_start{border-image: url(:/new/prefix1/pngwing.com.png) 0 0 0 0 stretch stretch; color : white;padding-bottom: 10px;}
#pushButton_start:pressed{color: black}
#pushButton_start:hover{border-style: outset; border-width: 2px; border-radius: 10px; border-color: beige;}
#pushButton_exit{border-image: url(:/new/prefix1/pngwing.com.png) 0 0 0 0 stretch stretch; color : white;padding-bottom: 10px;}
#pushButton_exit:pressed{color: black}
#pushButton_exit:hover{border-style: outset; border-width: 2px; border-radius: 10px; border-color: beige;}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="0" rowspan="2" colspan="4">
         <widget class="QFrame" name="frame">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="ClickableLabel" name="label_15">
           <property name="geometry">
            <rect>
             <x>740</x>
             <y>40</y>
             <width>111</width>
             <height>141</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="image.qrc">:/new/prefix1/НВ.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="ClickableLabel" name="label_18">
           <property name="geometry">
            <rect>
             <x>530</x>
             <y>310</y>
             <width>121</width>
             <height>121</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="image.qrc">:/new/prefix1/фотки людей/Angelina.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="ClickableLabel" name="label_16">
           <property name="geometry">
            <rect>
             <x>900</x>
             <y>20</y>
             <width>113</width>
             <height>150</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="image.qrc">:/new/prefix1/фотки людей/Inna.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="ClickableLabel" name="label_19">
           <property name="geometry">
            <rect>
             <x>590</x>
             <y>460</y>
             <width>121</width>
             <height>101</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="image.qrc">:/new/prefix1/фотки людей/Elina.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="ClickableLabel" name="label_20">
           <property name="geometry">
            <rect>
             <x>720</x>
             <y>550</y>
             <width>121</width>
             <height>101</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="image.qrc">:/new/prefix1/фотки людей/Lesha2.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="ClickableLabel" name="label_17">
           <property name="geometry">
            <rect>
             <x>620</x>
             <y>160</y>
             <width>91</width>
             <height>130</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="image.qrc">:/new/prefix1/фотки людей/Drozd.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_exit">
           <property name="geometry">
            <rect>
             <x>790</x>
             <y>420</y>
             <width>350</width>
             <height>140</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Segoe Print</family>
             <pointsize>17</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Выйти</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_start">
           <property name="geometry">
            <rect>
             <x>790</x>
             <y>300</y>
             <width>350</width>
             <height>140</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Segoe Print</family>
             <pointsize>17</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Начать игру</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_command">
           <property name="geometry">
            <rect>
             <x>780</x>
             <y>180</y>
             <width>350</width>
             <height>140</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Segoe Print</family>
             <pointsize>17</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhNone</set>
           </property>
           <property name="text">
            <string>Распределить 
 команды </string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRepeatInterval">
            <number>100</number>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="ClickableLabel" name="label_25">
           <property name="geometry">
            <rect>
             <x>910</x>
             <y>560</y>
             <width>121</width>
             <height>121</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="image.qrc">:/new/prefix1/фотки людей/Petrik.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="ClickableLabel" name="label_23">
           <property name="geometry">
            <rect>
             <x>1210</x>
             <y>460</y>
             <width>91</width>
             <height>101</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="image.qrc">:/new/prefix1/фотки людей/Lesha.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="ClickableLabel" name="label_24">
           <property name="geometry">
            <rect>
             <x>1230</x>
             <y>320</y>
             <width>131</width>
             <height>111</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="image.qrc">:/new/prefix1/фотки людей/Ksyusha.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="ClickableLabel" name="label_21">
           <property name="geometry">
            <rect>
             <x>1180</x>
             <y>180</y>
             <width>171</width>
             <height>131</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="image.qrc">:/new/prefix1/фотки людей/Pasha.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="ClickableLabel" name="label_22">
           <property name="geometry">
            <rect>
             <x>1050</x>
             <y>60</y>
             <width>141</width>
             <height>111</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="image.qrc">:/new/prefix1/фотки людей/Anya.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="ClickableLabel" name="label_26">
           <property name="geometry">
            <rect>
             <x>1070</x>
             <y>550</y>
             <width>131</width>
             <height>101</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="image.qrc">:/new/prefix1/фотки людей/Gunya.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="2">
         <widget class="QLabel" name="label">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Rockwell Condensed</family>
            <pointsize>40</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
            <underline>false</underline>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="pixmap">
           <pixmap resource="image.qrc">:/new/prefix1/title.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>clickablelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
